docs/deploy/installation.md

```diff
@@ -213,7 +213,7 @@ We recommend using the Helm chart to install the controller. The chart supports
             spec:
                 containers:
                     - args:
-                        - --cluster-name=<INSERT_CLUSTER_NAME>
+                        - --cluster-name=your-cluster-name
```

Comment:
It's not clear that "your-cluster-name" is a metasyntactic variable. The usual convention is to either use all-caps or surround with angle brackets. Elsewhere in this document metasyntactic variables are lowercase and surrounded by angle brackets.

```suggestion
                        - --cluster-name=<your-cluster-name>
```

Reply:
If we change it to `<your-cluster-name>` then we also need to update `INSERT_CLUSTER_NAME` 
https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/fd1bbbbfad42fc1b05fc71027242e2fcc4496c44/docs/deploy/installation.md?plain=1#L217C56-L217C56